frontend/e-learning-fe/src/components/notAdminUsers.js

```diff
@@ -0,0 +1,92 @@
+import React, { useEffect, useState } from "react";
+
+import API from "../api";
+
+function NotAdminUsers() {
+  const [users, setUsers] = useState([]);
+  const [loading, setLoading] = useState(false);
+  const token = localStorage.getItem("token");
+
+  const fetchData = async () => {
+    await API.notAdminUsers
+      .getNotAdminUsers({
+        token,
+      })
+      .then((data) => {
+        setUsers(data.data);
+        setLoading(true);
+      });
+  };
+
+  useEffect(() => {
+    fetchData();
+  }, [loading]);
+
+  return (
+    <div>
+      {loading ? (
```

Comment:
Yes sir all of my implementations are the same, if `isLoading` is false walang display but if true meron na syang idisplay

Reply:
please change it since it is counter intuitive hence confusing when read in code. It should display data when `isLoading` is false
```
isLoading ? 'do not display anything' : 'display data'
```